feat(app): option to build the app with maven wallet-sdk. (#414)
trustbloc/wallet-sdk: demo/app/README.md

  - Open the app in XCode
  - Install app to Simulator or Device from the XCode Build tab

> ### Build with WalletSDK from the package manager
> - Android
>   - Create GitHub Personal access tokens (classic)
>   - run `WALLET_SDK_USR=GIT_USR WALLET_SDK_TKN=TKN flutter run`
>   - or edit `android/local.properties` and add there `wallet-sdk-pkg.usr` and `wallet-sdk-pkg.tkn` variables and run `flutter run`
>   - if you are switching between the package manager version and the local version, often Gradle cache need to be cleared `rm -r $HOME/.gradle/`
> 

Reviewer: ```suggestion
### Build with WalletSDK from Maven
- Android
  - Create GitHub personal access tokens (classic).
  - Run `WALLET_SDK_USR=GIT_USR WALLET_SDK_TKN=TKN flutter run`. Alternatively, add `wallet-sdk-pkg.usr` and `wallet-sdk-pkg.tkn` variables to `android/local.properties` and then run `flutter run`.
 
 Note: If you are switching between the Maven version and a local version, the Gradle cache may need to be cleared. To do this, run `rm -r $HOME/.gradle/`.

```

Some formatting/writing suggestions.

Author: Thanks!
